pages/guide.py

```diff
@@ -354,6 +351,11 @@ def process_post_data(self, feature_id, stage_id=0):
     if self.touched('i2s_lgtms'):
       feature.i2s_lgtms = self.split_emails('i2s_lgtms')
 
+    if self.touched('next_review_date'):
+      date_str = self.form.get('next_review_date')
+      date_val = datetime.date.fromisoformat(date_str)
```

Comment:
Maybe have a test for it so we know the format of `date_str`?

Reply:
Added.